liota/lib/utilities/utility.py

```diff
@@ -272,6 +273,16 @@ def read_liota_config(section, name):
         log.warn('liota.conf file missing')
     return value
 
+def read_user_config(config_file_path):
+
+    config = ConfigParser.RawConfigParser()
+    config.optionxform = str
+    config.read(config_file_path)
+
+    user_config = dict(config.items('DEFAULT'))
```

Comment:
Does it take care of getting the values from other sections in sampleprop.conf like 

IOTCC
GENERICMQTT